skymp5-server/cpp/addon/main.cc

```diff
@@ -1254,6 +1254,10 @@ void ScampServer::RegisterChakraApi(std::shared_ptr<JsEngine> chakraEngine)
         res = arr;
       } else if (propertyName == "isDisabled") {
         res = JsValue(refr.IsDisabled());
+      } else if (propertyName == "isDead") {
+        if (auto actor = dynamic_cast<MpActor*>(&refr)) {
+          res = JsValue(actor->IsDead());
```

Comment:
JsValue::Boolean(actor->IsDead()) would be better
I remember problems on JsEngine side with constructor from bool